cloud/aws/ses/mode/emails.pm

```diff
@@ -0,0 +1,240 @@
+#
+# Copyright 2020 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::aws::ses::mode::emails;
+
+use base qw(centreon::plugins::templates::counter);
+
+use strict;
+use warnings;
+
+my %metrics_mapping = (
+    'Send' => {
+        'output' => 'number of successful calls',
```

Comment:
Think that it will be used as the output message. 

If the send value is 4 it will gives " 4 number of successful calls". Maybe "emails sent" would be enough  ? wdyt ?

Reply:
Yeah that's better, I will change it.

Reply:
Ok let's do it :)